src/components/Navbar.js

```diff
@@ -1,15 +1,42 @@
-import React from "react"
+import React, { Component } from "react"
 import { Link } from "react-router-dom"
+import "./Navbar.css"
 
-const Navbar = () => {
-    return (
-        <div className="Navbar">
-            <li><Link to="/excursions">Excursion</Link></li>
-            <li><Link to="/transport">Transportation</Link></li>
-            <li><Link to="/contact">Contact</Link></li>
-            <li><Link to="/signup">Sign up</Link></li>
-        </div>
-    )
-}
+class Navbar extends Component {
+    constructor() {
+        super()
+        this.state = {
+            user: "organizer"    // For the displayed menu items depend on this 'user' state.
+        }
+    }
+
+    render() {
 
+        if (this.state.user == "none") {
+            return (<div className="Navbar">
+                <Link to="/">Start</Link>
+            </div>)
```

Comment:
Another way of doing this is to redirect to a login screen if there is no user. You're on the right path thinking in conditional rendering, but it looking into how other websites are doing it might inspire you. Try to log out of your favourite sites and see how it changes.

== compared non-strictly. Unless you have a good reason for not doing it, use === by default.

Instead of user = "none" you can have user = false or null. Then you can test if (user) instead.